src/viewport.js

```diff
@@ -366,9 +378,10 @@ export class Viewport {
 
   /** @private */
   resize_() {
-    let oldWidth = this.width_;
-    this.width_ = this.getSize().width;
-    this.changed_(oldWidth != this.width_, 0);
+    let oldSize = this.size_;
+    this.size_ = null;  // Need to recalc.
```

Comment:
Got it. Missed it. I think doing `this.size_ = this.binding_.getSize();` here would be a bit safer. But I don't feel to strongly about this.